plat/arm/css/sgm/sgm_plat_config.c

```diff
@@ -67,3 +67,13 @@ css_plat_config_t *get_plat_config(void)
 	assert(css_plat_info != NULL);
 	return css_plat_info;
 }
+
+#if TRUSTED_BOARD_BOOT && LOAD_IMAGE_V2
+int plat_get_mbedtls_heap(void **heap_addr, size_t *heap_size)
+{
+	assert(heap_addr != NULL);
```

Comment:
The #ifs are needed because:
1) both the plat_get_... as well as the arm_get_... functions depend on Mbed TLS sources which are included only if TRUSTED_BOARD_BOOT is used,
2) if we don't have LOAD_IMAGE_V2 there is no way to pass the heap address/size from BL1 to BL2. As such, we would crash in BL2.

Regarding the asserts, it's more like additional safety really. Each function (plat_... and arm_...) should be self-contained so that's why the asserts were included in both.